lib/egauge/power_helpers.rb

```diff
@@ -10,6 +10,21 @@ def monthly_kwh(number_of_months = 12)
       query(:C => nil, :T => months(number_of_months))
     end
 
+    def past_24_hrs_kwh(csv=false)
+      end_time = Time.now.beginning_of_hour
+      ar = []
+      24.times do |i|
+        ar << (end_time - i.hours).to_i
+      end
+
+      options = {T: ar.join(",")}
+      if csv
+        csv_query(options)
```

Comment:
I like how these are separate public methods on `Client`. For this helper though, I think I would just change the param to either be a keyword argument or to take a symbol of the method (i.e. `:csv_query`) just to make it more clear what the param is doing.